video/profiles.go

```diff
@@ -208,7 +208,7 @@ type OutputVideoFile struct {
 }
 
 func PopulateOutput(requestID string, probe Prober, outputURL string, videoFile OutputVideoFile) (OutputVideoFile, error) {
-	outputVideoProbe, err := probe.ProbeFile(requestID, outputURL)
+	outputVideoProbe, err := probe.ProbeFile(requestID, outputURL, "-analyzeduration", "15000000")
```

Comment:
@thomshutt you saved me a few hours of debug. Probing fmp4 files were failing with cryptic error messages until I saw your other PR :)